src/components/uploads/ImageUploader.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useRef } from "react";
+
+export const defaultImg =
```

Comment:
상수로 쓰이는 것 같아서 저는 대문자로 하면 더 좋을 것 같아요..!

```suggestion
const DEFAULT_IMAGE =
```

Reply:
사실 이 부분은 constant 폴더를 만들어 그 안에서 이미지 관련해 상수를 관리하고 싶더라고요!(이때 대문자 관리)
아직 저희가 폴더 구조에 대해서 이야기 나눈 바가 없어 테스트용으로 이렇게 해놓았는데 우선은 대문자로만 수정하겠습니다! 좋은 지적 감사합니다 :)